src/v/kafka/server/group_metadata_migration.cc

```diff
@@ -700,7 +701,11 @@ ss::future<> group_metadata_migration::start(ss::abort_source& as) {
                   mlog.info,
                   "kafka_internal/group topic does not exists, activating "
                   "consumer_offsets feature");
-                co_await do_activate_feature(_as);
+                try {
+                    co_await do_activate_feature(_as);
+                } catch (...) {
+                    co_return;
```

Comment:
why we just return on exception ? This should retry

Reply:
Hm, my fault.